core-java-modules/core-java-string-conversions-2/src/test/java/com/baeldung/stringtocamelcase/StringToCamelCaseUnitTest.java

```diff
@@ -0,0 +1,103 @@
+package com.baeldung.stringtocamelcase;
+
+
+import org.junit.Test;
+
+import static com.baeldung.stringtocamelcase.StringToCamelCase.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.Assert.assertEquals;
+
+
+public class StringToCamelCaseUnitTest {
```

Comment:
As this is JUnit5, the `public` modifier should not appear on the class or its methods

Reply:
Note: this is not yet resolved

Reply:
I believe this is junit4. It will break if I don't put the public modifiers.

Reply:
Yes. This is now JUnit 4. Perhaps it was JUnit 5 before. Sorry about the confusion.